### ISSUE TITLE
`new super()` in static method does not cause error

### ISSUE THREAD

**Issue by Withered-Flower-0422** [microsoft/TypeScript#63451]
### 🔎 Search Terms

super

### 🕗 Version & Regression Information

This exists in ts6.0

### 💻 Code

```ts
class A extends class {} {
    static {
        new super()
    }

    static f() {
        new super()
    }
}
```

### 🙁 Actual behavior

no error

### 🙂 Expected behavior

`new super()` is invalid and should throw error.

**Comment by RyanCavanaugh** [comment]
🤖 Thank you for your issue! I've done some analysis to help get you started. This response is automatically generated; feel free to 👍 or 👎 this comment according to its usefulness.

## Similar Issues

Here are the most similar issues I found
- (69%) [microsoft/typescript#60669](https://redirect.github.com/microsoft/typescript/issues/60669): Error "TS2335" false positive
- (67%) [microsoft/typescript#7492](https://redirect.github.com/microsoft/typescript/issues/7492): Support super calls in static functions
- (67%) [microsoft/typescript#44515](https://redirect.github.com/microsoft/typescript/issues/44515): super reference in private method throws 
- (66%) [microsoft/typescript#54054](https://redirect.github.com/microsoft/typescript/issues/54054): The child class should not be able to access the class field defined by the parent class via super
- (66%) [microsoft/typescript#718](https://redirect.github.com/microsoft/typescript/issues/718): Inheritance of static methods is broken

If your issue is a duplicate of one of these, feel free to close this issue. Otherwise, no action is needed.

## Possibly Working as Intended

It looks like this behavior might be intentional. Here's what a computer had to say about it.



Inside a static context (a static method or a static initialization block) the identifier `super` is an expression that evaluates to the constructor function of the base class. Because it is an ordinary value there, you can do anything with it that you can do with a normal constructor function, including constructing it with `new`:

```ts
class Base {}
class Derived extends Base {
  static make() {
    return new super();   // same as: return new Base();
  }
}
```

What is *not* allowed is `super()`—a direct *call* to the superclass constructor—outside of a derived class’s constructor. But `new super()` falls under the normal `new <expression>()` grammar and is valid JavaScript. Because the code is valid according to the ECMAScript spec, TypeScript correctly accepts it and emits corresponding JavaScript.

**Comment by Withered-Flower-0422** [comment]
No.
Javascript never allows `new super()`.
In both browser and node runtime, `new super()` just throws errors.

**Comment by mkantor** [comment]
Why did the bot add the "Fix Available" label? I can't find any relevant pull requests.

**EDIT:** Seems like this happened to #63391 as well.

**Comment by RyanCavanaugh** [comment]
Bot's broken and no one has had the time to figure out why 😅. We don't really use the label for anything at the moment
